exercise/views.py

```diff
@@ -477,7 +477,7 @@ def get(self, request, *args, **kwargs):
             return response
 
         if self.file.is_passed():
-            return HttpResponse(bytedata, content_type=self.file.get_mime())
+            return HttpResponse(bytedata, content_type="application/octet-stream")
```

Comment:
Let's keep the old version of the code here so that the mime type is correct for the commonly used file types (assuming that their filename extension is correct). No need to force everything to octet-stream here.

Reply:
Done.